### ISSUE TITLE
understanding download creation and timing

### ISSUE THREAD

**Issue by punkish** [plazi/treatmentBank#22]
@gsautter 

You explained this to me before but I am confused still… please clarify for me how the creation and naming of the zip archives work. If I look today at the dumps (https://tb.plazi.org/dumps/) I see the following

| Filename | Description | Size | Last Modified |
|---------- | ----------- |----- |--------------|
|plazi.zenodeo.zip | Simplified Plain XML | 2548608 kb | Sat, 01 Jan 2022 02:00:00 GMT+0000|
|plazi.zenodeo.monthly.zip | Simplified Plain XML (updates since plazi.zenodeo.zip) | 65 kb | Sun, 02 Jan 2022 02:00:00 GMT+0000|

Since it has been only day since the full archive was created, why is the second archive called "monthly" and not "daily"? From what you explained to me, I would have expected there to be a daily archive for 6 days (from full archive creation), then on the 7th day, a weekly archive would have been created, then on the 30 days there would be a monthly archive… what's going on?

**Comment by gsautter** [comment]
You're right, this turn-of-the-year behavior may seem a bit counter intuitive at the beginning ... however, there's a simple idea behind this: the finer granularities don't exist without the coarser ones. In particular, it means this:  
- if there is no `monthly`, it implies there is no `weekly` or `daily`
- if there is no `weekly`, it implies there is no `daily`

This allows for a very simply download/import logic:
- check the full dump, and download it if your local version has an earlier timestamp
- check the `monthly`:
  - if there is none, you're done right here
  - download it if your local version doesn't exist or has an earlier timestamp
- check the `weekly`:
  - if there is none, you're done right here
  - download it if your local version doesn't exist or has an earlier timestamp
- check the `daily`, and download it if your local version doesn't exist or has an earlier timestamp

In particular, it saves you even checking for the `daily` if you don't find the `weekly`, etc.

The `weekly` and `daily` will become available in the next couple of days, and then things will go back to their normal course.

If you have a different idea how the naming logic should behave, please let me know ... the current one is simply what I figured might be easiest to handle for client code (using above logic) ... and I might well be wrong ...
